bears/c_languages/ClangBear.py

```diff
@@ -52,17 +52,17 @@ def diff_from_clang_fixit(fixit, file):
     return Diff.from_string_arrays(file, new_file)
 
 
-def sourcerange_from_clang_range(range):
+def sourcerange_from_clang_range(srcrange):
```

Comment:
It would be better to call the parameter as `clang_range` rather than `srcrange`.

Reply:
done @MacBox7